Resources/ServerInfo/Guidebook/Mobs/_DEN/Ovinia.xml

```diff
@@ -0,0 +1,26 @@
+<Document>
+  # Ovinia
+  (lore written under this section is provided by 2blahajs on Discord or juniwoofs on GitHub)
```

Comment:
When I said "move this down" I was referring to the line of "lore written under this section", which is currently in a section without lore